### PR TITLE
fixes #10789 with new banned methods

### LINKED ISSUE #10789: update conformance config with banned methods
per @jridgewell 

Object.values, Object.entries, Object.getOwnPropertyDescriptors, String#padStart, String#padEnd 
#banemall

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

File: build-system/conformance-config.textproto [hunk 1]
```diff
@@ -60,6 +60,36 @@ requirement: {
 }
 
 requirement: {
+  type: BANNED_PROPERTY_CALL
+  error_message: 'string.prototype.padStart is not allowed'
+  value: 'string.prototype.padStart'
```
Comment (jridgewell) [inline build-system/conformance-config.textproto]: Already above.

File: build-system/conformance-config.textproto [hunk 2]
```diff
@@ -60,6 +60,36 @@ requirement: {
...
+}
+
+requirement: {
+  type: BANNED_PROPERTY_CALL
+  error_message: 'string.prototype.padEnd is not allowed'
+  value: 'string.prototype.padEnd'
```
Comment (jridgewell) [inline build-system/conformance-config.textproto]: Already above.

File: build-system/conformance-config.textproto (hunk 1 above)
Comment (erwinmombay) [reply build-system/conformance-config.textproto]: fixed.

File: build-system/conformance-config.textproto (hunk 2 above)
Comment (erwinmombay) [reply build-system/conformance-config.textproto]: fixed.

**Pull request merged**